src/app/lib/config/services/yoruba.js

```diff
@@ -74,7 +74,7 @@ export const service = {
       error: {
         404: {
           statusCode: '404',
-          title: 'A kò rí ojú ewé náà',
+          title: 'A kò rí ojú ewé náà',
```

Comment:
What's changed here?

Reply:
I am not sure why GitHub's flagged this.